cli/src/tasks/sourcemaps.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  readdirSync,
+  existsSync,
+  readFileSync,
+  writeFileSync,
+  unlinkSync,
+} from '@ionic/utils-fs';
+import { join, extname } from 'path';
+
+import type { Config } from '../definitions';
+import { logger } from '../log';
+
+function findJSAssetsDir(buildDir: string): string {
```

Comment:
JS files could be any folder. Im not sure you would want to hard code this for React and Vue.